### ISSUE TITLE
Thank you 💜 

### ISSUE THREAD

**Issue by yunalebt** [vladciobanu/vladciobanu.github.io#2]
Hi Evie, 

Thank you very much for your post "Always Knew". I found myself in parts of your post very well. Thank you so much for sharing your thoughts. 

I'm still very new on my journey; I've only had my Coming-Out to my family and friends a week ago. My family has been consistently positive about it; my wife too. She just needs a little time to get used to everything new, but it will be fine.

Can you be found in the Fediverse? I would like to follow you there. I don't use Twitter; there's too much hate.  If you want to follow me: https://troet.cafe/web/@yuna (troet.cafe is just one of the various instances of how to get into the Fediverse via Mastodon (Twitter alternative)). However, I write mainly in German; nevertheless, I would be very happy to get in touch with you.

Many greetings from Northern Germany :-)

Yuna

Translated with www.DeepL.com/Translator (free version)

**Comment by eviefp** [comment]
Hey, I'm sorry but I just saw your message now. I missed this notification.

I hope everything is going well for you!

I am indeed on mastodon now, it's @vie@hachyderm.io ; feel free to reach out!

**Comment by 0xameer** [comment]
I have followed - how I nix [1]
It ,sort of, works here, but it ll be neater if you specify $USER and $PATH of nixfiles
in plain English, which user has to create nixfiles and which $home  , /root or /home/username
the latter - perm issues 
the former  - path issues , despite adding hm channel
2c 
[1] https://eevie.ro/posts/2022-01-24-how-i-nix.html
